api/src/main/java/org/openmrs/module/aijar/AijarActivator.java

```diff
@@ -104,6 +104,9 @@ public void started() {
             appFrameworkService.disableApp("coreapps.visitByEncounterType");
             appFrameworkService.disableApp("coreapps.dataIntegrityViolations");
             
+            // enable apps
+            appFrameworkService.disableApp("coreapps.relationships");
```

Comment:
@ssmusoke The comment and the function don't match. comment is enable, the method call is to disable.